src/TAlias.cpp

```diff
@@ -299,10 +305,30 @@ void TAlias::setRegexCode( QString code )
     if( re == NULL )
     {
         mOK_init = false;
+        if( mudlet::debugMode ) {
+            TDebug( QColor(Qt::white), QColor(Qt::red) ) << "REGEX ERROR: failed to compile, reason:\n"
+                                                         << error
+                                                         << "\n"
+                                                         >> 0;
+            TDebug( QColor(Qt::red), QColor(Qt::gray) ) << "in: \""
+                                                        << mRegexCode
+                                                        << "\"\n"
+                                                        >> 0;
+        }
+        setError( tr( "Error: in \"Pattern:\", faulty regular expression, reason: \"%1\"." )
```

Comment:
I think I was seeing the `<b><font color='blue'>`...`</font></b>` (when the tags are correctly ordered) from `bool TLuaInterpreter::compile(...)` showing up as a __literal part__ (i.e. reproduce verbatim and not interpreted as markup) of the error message in the Central Debug Console. Some further research/work/fixing may be needed before I turn the air (or rather, the error message) blue...